server/controllers/routes/user/cohort/getSpecificCohort.js

```diff
@@ -0,0 +1,10 @@
+const {
+  cohort: { getSpecificCohort },
+} = require('../../../../database/queries');
+
+exports.getSpecificCohort = (req, res) => {
+  getSpecificCohort(req.params.cohortid)
+    .then(({ rows }) => res.json(rows))
+    // eslint-disable-next-line no-console
+    .catch((err) => console.error(err));
```

Comment:
don't disable eslint and try handle error
```suggestion
    .catch((err) => next(err));
```